examples/lb3-application/README.md

```diff
@@ -95,10 +95,10 @@ applications can use them.
 
 Start the new LB4 application
 
-    ```sh
-    $ npm start
-    Server is running at http://127.0.0.1:3000
-    ```
+   ```sh
+   $ npm start
+   Server is running at http://127.0.0.1:3000
+   ```
```

Comment:
Why not remove the indentation altogether?

````suggestion
```sh
$ npm start
Server is running at http://127.0.0.1:3000
```
````